index.html

```diff
@@ -3363,6 +3363,13 @@ <h3>JSON-LD</h3>
 Model is available at <code>https://www.w3.org/2018/credentials</code>.
         </p>
 
+        <p>
+This specification restricts the usage of JSON-LD representations. JSON-LD <a
+href="https://www.w3.org/TR/json-ld/#compacted-document-form">compact document
+form</a> MUST be utilized for all expressions of the data model described by
```

Comment:
what is the difference between "expression" and "representation" ?

Reply:
Historically, I think we've tried to use "expression" to talk about something in the abstract (the data model) and "representation" to talk about something concrete (a serialization). I mixed them up here, and will change both to speak to "representation" -- does that work for you, @OR13 ?